community-stabilization-fund-api/pages/donate/donate.module.css

```diff
@@ -1,21 +1,24 @@
 .donate {
-  text-align: center;
   margin-top: 12rem;
 }
 
 .donate p {
+  text-align: center;
```

Comment:
Good more specific styling!

Reply:
Understood, we can plan for a more specific rewrite of css going forward.